rsp_jupyter_extensions/handlers/execution.py

```diff
@@ -37,16 +41,46 @@ def post(self) -> None:
         **Request headers.**
         Set the ``X-Kernel-Name`` header to the name of the kernel to use for
         execution.
+
+        Set the ``X-Clear-Local-Site-Packages`` header to "true" (in any case)
+        in order to remove any locally-installed packages prior to notebook
+        execution.
         """
         input_str = self.request.body.decode("utf-8")
         kernel_name = self.request.headers.get("X-Kernel-Name", None)
+        clr_pkg_hdr = self.request.headers.get(
+            "X-Clear-Local-Site-Packages", ""
+        ).lower()
+        clr_pkg = False
+        if clr_pkg_hdr == "true":
```

Comment:
```suggestion
        clr_pkg_hdr = self.request.headers.get(
            "X-Clear-Local-Site-Packages", ""
        ).lower()
        clr_pkg = False
        if self.request.headers.get(
            "X-Clear-Local-Site-Packages", ""
        ).lower().strip() == "true":
            do_remove_local_packages = True
```

So, extra processing, and also stop abbreviating variable names so people can understand them.